priv/site/schedule.slime

```diff
@@ -0,0 +1,134 @@
+---
+layout: _layout.slime
+---
+
+.Schedule
+  = include(@env, "_includes/schedule/nav.slime")
+  .Schedule-hero
+    .Schedule-heroImage
+      img src="public/schedule_hero_image.jpg"
+
+    .Schedule-content
```

Comment:
This is a nit pick, but it seems to me that since this is specific to the hero, a better name would be something like `Schedule-heroContent` or something along those line. What do you think?